readme.md

```diff
@@ -1,88 +1,122 @@
 # fast-safe-stringify
 
-Safely and quickly serialize JavaScript objects
+Safe and fast replacement serialization replacement for [JSON.stringify][].
 
-Detects circular dependencies instead of throwing (as per usual `JSON.stringify`
-usage)
+Gracefully handles circular structures instead of throwing.
+
+Provides a stable version as well that will also gracefully handle circular
```

Comment:
"stable version" is ambiguous (immediately makes you think software versioning)

how about

Provides additional optional sorted key functionality (stable sorted JSON), which is also fast and safe